gslab_scons/log.py

```diff
@@ -60,8 +60,11 @@ def end_log(log = 'sconstruct.log', excluded_dirs = []):
                 sconstruct.write(sconscript.read())
 
     # move top level logs to /release/ directory.
+    release_dir = './release/'
```

Comment:
Can we make `release_dir = './release/'` as an argument for this function instead of writing this inside the function?